docs/Testing.rst

```diff
@@ -489,3 +489,53 @@ code that uses an ``if true {}`` or similar no-op scope instead of
 If you're specifically testing the autoreleasing behavior of code, or do not
 expect code to interact with the Objective-C runtime, it may be OK to use ``if
 true {}``, but those assumptions should be commented in the test.
+
+libFuzzer Integration
```

Comment:
@jrose-apple yes, but I couldn't think of a better place. Do you think it should be just a separate `.rst` file? Also fuzz-testing standard library does sound like a good idea.